capstone/src/main/webapp/cover.css

```diff
@@ -187,3 +187,38 @@ hr {
   opacity: 1;
   transform: translateX(-15%) translateY(0px);
 }
+
+/* Attribute Dropdown */
+.checkbox-menu li label {
+  display: block;
+  padding: 3px 10px;
+  clear: both;
+  font-weight: normal;
+  line-height: 1.42857143;
```

Comment:
This is a very specific number, any reason 😆

Reply:
It's from the Bootstrap website - I'll change it to 1.5 to make things nice and round!